### PR TITLE
updated docker images list for docker-compose + fixed jaeger, grafana…

### PR DESCRIPTION
**Pull request by bhogayatakb**
1. Replaced Docker images in docker compose
2. Fixed Jaeger, Grafana setup with port-forwarding
`http://localhost:16686/jaeger/ui/search`
`http://localhost:3000/grafana`|
3. Updates to run Next.js APM with frontend project

Now, anyone should be able to run the project with docker-compose + MIddleware APMs using this command:
`make start`

### CODE REVIEW

**Review by sanjoyment — APPROVED** [review]

File: src/checkoutservice/main.go [hunk 1]
```diff
@@ -210,19 +210,19 @@ func (cs *checkoutService) Watch(req *healthpb.HealthCheckRequest, ws healthpb.H
...
-		if err != nil {
-			span.AddEvent("error", trace.WithAttributes(attribute.String("exception.message", err.Error())))
-		}
-	}()
+	// span := trace.SpanFromContext(ctx)
+	// span.SetAttributes(
+	// 	attribute.String("app.user.id", req.UserId),
+	// 	attribute.String("app.user.currency", req.UserCurrency),
+	// )
+	log.Infof("[PlaceOrder22] user_id=%q user_currency=%q", req.UserId, req.UserCurrency)
```
Comment (parth-dadhaniya) [inline src/checkoutservice/main.go]: why `PlaceOrder22` ?

File: src/frontend/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "npm run grpc:generate && NODE_OPTIONS='-r ./utils/telemetry/Instrumentation.js' next dev",
     "build": "next build",
-    "start": "node --require ./Instrumentation.js server.js",
+    "start": "node --require ./instrumentation.js server.js",
```
Comment (parth-dadhaniya) [inline src/frontend/package.json]: IMO this is not needed.

File: src/otelcollector/otelcol-config.yml [hunk 3]
```diff
@@ -13,7 +23,20 @@ receivers:
...
+  otlp/jaeger:
+    endpoint: "jaeger:4317"
+    tls:
+      insecure: true
   logging:
+    loglevel: debug
+  otlp/3:
+    endpoint: ${MW_TARGET}
+    headers:
+      authorization:  ${MW_API_KEY}
```
Comment (parth-dadhaniya) [inline src/otelcollector/otelcol-config.yml]:
```suggestion
      authorization: ${MW_API_KEY}
```

File: src/checkoutservice/main.go (hunk 1 above)
Comment (bhogayatakb) [reply src/checkoutservice/main.go]: Resolved. Was added for testing if changes are being reflected.

File: src/otelcollector/otelcol-config.yml (hunk 3 above)
Comment (bhogayatakb) [reply src/otelcollector/otelcol-config.yml]: Spacing Resolved.

File: src/frontend/package.json (hunk 2 above)
Comment (bhogayatakb) [reply src/frontend/package.json]: Reverted.

File: src/frontend/instrumentation.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const tracker = require('@middleware.io/agent-apm-nextjs');
+
+function register() {
+    tracker.track({
+        serviceName: "frontend-project-nextjs-apm-keval",
+        accessToken: "murwpaubimiwrfrtoxbwpoiuoztzdjgmsyph",
```
Comment (parth-dadhaniya) [inline src/frontend/instrumentation.js]: IMO, we should change this.

Comment (parth-dadhaniya) [reply src/frontend/instrumentation.js]: Should not use personal names.

File: src/frontend/utils/telemetry/Instrumentation.js [hunk 5]
```diff
@@ -1,41 +1,15 @@
...
-    gcpDetector,
-  ],
-});
+        // target: "https://spbnu.middleware.io:443",
+    });
+    tracker.warn("Instrumentation file scanned successfully!");
+}
 
-sdk.start();
+module.exports = { register };
```
Comment (parth-dadhaniya) [inline src/frontend/utils/telemetry/Instrumentation.js]: add new line char at end of the file.

File: src/frontend/utils/telemetry/Instrumentation.js [hunk 6]
```diff
@@ -1,41 +1,15 @@
...
-const {OTLPMetricExporter} = require('@opentelemetry/exporter-metrics-otlp-grpc');
-const {PeriodicExportingMetricReader} = require('@opentelemetry/sdk-metrics');
-const {alibabaCloudEcsDetector} = require('@opentelemetry/resource-detector-alibaba-cloud');
-const {awsEc2Detector, awsEksDetector} = require('@opentelemetry/resource-detector-aws');
-const {containerDetector} = require('@opentelemetry/resource-detector-container');
-const {gcpDetector} = require('@opentelemetry/resource-detector-gcp');
-const {envDetector, hostDetector, osDetector, processDetector} = require('@opentelemetry/resources');
+function register() {
+    tracker.track({
+        serviceName: "frontend-project-nextjs-apm-keval",
```
Comment (parth-dadhaniya) [inline src/frontend/utils/telemetry/Instrumentation.js]:
```suggestion
        serviceName: "frontend-project-nextjs-apm",
```
Should not use personal details.

File: src/frontend/utils/telemetry/Instrumentation.js (hunk 5 above)
Comment (bhogayatakb) [reply src/frontend/utils/telemetry/Instrumentation.js]: added

File: src/frontend/utils/telemetry/Instrumentation.js (hunk 6 above)
Comment (bhogayatakb) [reply src/frontend/utils/telemetry/Instrumentation.js]: updated.

**Review by parth-dadhaniya — APPROVED** [review]

**Pull request merged**